easybuild/easyconfigs/e/ega-quickview/ega-quickview-20240620-GCC-12.3.0.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'ega-quickview'
+version = '20240620'
+local_commit = 'fe2034d'
+
+homepage = 'https://github.com/EGA-archive/ega-quickview'
+description = """EGA-QuickView is a FUSE file system to access EGA files remotely."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+github_account = 'EGA-archive'
+source_urls = [GITHUB_SOURCE]
+sources = [{
+    "download_filename": "%s.tar.gz" % local_commit,
+    "filename": "%(name)s-%(version)s.tar.gz",
+}]
+checksums = ['90836e42009736a8e20a2569918638f3cb2b53574265ca4f4bed7abd81a5e887']
+
+builddependencies = [
+    ('make', '4.4.1'),
+    ('Autoconf', '2.71'),
```

```suggestion
    ('Autotools', '20220317'),
```

```
Can't exec "aclocal": No such file or directory at /project/def-maintainers/boegelbot/rocky9/zen3/software/Autoconf/2.71-GCCcore-12.3.0/share/autoconf/Autom4te/FileUtils.pm line 274.
autoreconf: error: aclocal failed with exit status: 2
```
`aclocal` is provided by `Automake` - so we should provide all of the `Autotools` here.